v2/terraform_flannel.py

```diff
@@ -198,30 +202,34 @@ def _waitForConnection(self, machine, windows):
         out, _, ret = utils.run_cmd(cmd, stdout=True, cwd=self.ansible_playbook_root, shell=True)
         return ret, out
 
-    def _copyTo(self, src, dest, machine, windows=False, root=False):
-        self.logging.info("Copying file %s to %s:%s." % (src, machine, dest))
+    def _copyTo(self, src, dest, machines, windows=False, root=False):
+        self.logging.info("Copying file %s on %s to %s." % (src, machines, dest))
         cmd = ["ansible"]
         if root:
             cmd.append("--become")
         if not windows:
             cmd.append("--key-file=%s" % self.opts.ssh_private_key_path)
-        cmd.append(machine)
+
+        vms = " "
+        vms = "'" + vms.join(machines) + "'"
+        cmd.append(vms)
+
         cmd.append("-m")
         module = "win_copy" if windows else "copy"
         cmd.append(module)
         cmd.append("-a")
         cmd.append("'src=%(src)s dest=%(dest)s flat=yes'" % {"src": src, "dest": dest})
 
-        ret, _ = self._waitForConnection(machine, windows=windows)
+        ret, out = self._waitForConnection(machines, windows=windows)
         if ret != 0:
-            self.logging.error("No connection to machine: %s", machine)
-            raise Exception("No connection to machine: %s", machine)
+            self.logging.error("No connection to %s. Error: %s" % (machines, out))
```

Comment:
Since we don't no specifically which machine we are talking about here, there's no need to print the whole list. Enough to say: "No connection to machines. Error: %s" % out